doc/1.4/language.md

```diff
@@ -2823,6 +2823,19 @@ where each input parameter is of [*serializable type*](#saved-variables).
 This means that `after` statements cannot be used with methods
 that e.g. have pointer parameters.
 
+Any events posted via an `after` statement are associated with the object
+that the method using the statement belongs to. It is possible to cancel
```

Comment:
The "Any events..." sentence is hard to parse, but indeed correct. My eyes initially parsed "the statement belongs to" as a unit. Maybe insert ", i.e., the caller," somewhere?

Reply:
I don't feel like "the caller" works. After statements are not methods/functions, and do not have a caller -- but more concerning, "the caller" could be confused with "the caller of _the method_", which is very incorrect.

I was considering:
```
Any events posted via an `after` statement are associated with the object
that contain the method using the statement. 
```
But I was worried that might've caused confusion with regards to methods defined in templates, especially `shared` methods.

Reply:
I liked the 'contains' sentence better, I don't see how it changes the risk of confusion. I like 'contain' better than 'use'; I think the repetition in 'object that contains the method that contains the statement' is fine since the containment is indeed transitive.